src/util/profile/profile.ts

```diff
@@ -25,14 +25,15 @@ export interface UpdatableProfile {
 }
 
 export interface Profile extends UpdatableProfile {
-  readonly accessToken: Promise<TokenValueType>;
+  readonly accessToken: Promise<string>;
```

Comment:
I'll check again but I ran into compiler issues with TS 2.1 which apparently has better type checking for generics and errored in this file.